.buildkite/pull-requests.json

```diff
@@ -95,6 +95,38 @@
             "skip_target_branches": [ ],
             "skip_ci_on_only_changed": [ ],
             "always_require_ci_on_changed": [ "^deploy/kubernetes/.*", ".buildkite/deploy/kubernetes/.*", "^libbeat/docs/version.asciidoc"]
+        },
+        {
+            "enabled": true,
+            "pipelineSlug": "beats-libbeat",
+            "allow_org_users": true,
+            "allowed_repo_permissions": ["admin", "write"],
+            "allowed_list": [ ],
+            "set_commit_status": true,
+            "build_on_commit": true,
+            "build_on_comment": true,
+            "trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:test)\\W+(?:libbeat))|^/test libbeat$",
+            "always_trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:test)\\W+(?:libbeat))|^/test libbeat$",
```

Comment:
**nit**: maybe just `^/test libbeat$` for the shake of simplicity?

Reply:
fixed